imports/plugins/core/accounts/client/helpers/accountsHelper.js

```diff
@@ -32,8 +39,9 @@ export function sortGroups(groups) {
 }
 
 /**
- * getInvitableGroups - helper - client
- * @summary This generates a list of groups the user can invite to.
+ * @method getInvitableGroups
+ * @memberof Accounts
+ * @summary helper - client This generates a list of groups the user can invite to.
```

Comment:
Typo here that makes it hard to understand what the function does? (inherited, but maybe we can fix it while we notice it?)

Reply:
@impactmass we may need to update the first line of this jsdoc summary to be a little easier to understand in the docs context

Reply:
Okay. I'll make a note of that for next PR